build.gradle

```diff
@@ -17,6 +17,13 @@ dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jre8"
     compile "org.jetbrains.kotlinx:kotlinx-coroutines-core:0.18"
     compile "com.fasterxml.jackson.module:jackson-module-kotlin:2.9.0"
+    compile 'io.arrow-kt:arrow-core:0.6.1'
+    compile 'io.arrow-kt:arrow-typeclasses:0.6.1'
+    compile 'io.arrow-kt:arrow-data:0.6.1'
+    compile 'io.arrow-kt:arrow-instances:0.6.1'
+    compile 'io.arrow-kt:arrow-syntax:0.6.1'
+    kapt    'io.arrow-kt:arrow-annotations-processor:0.6.1'
```

Comment:
Features are still poorly documented, can we illustrate this FP pattern without external dependency?

Reply:
The amount of code needed to be able to do replicate the same behaviour could be considered a completely project/exercise.

Even if the doc in lacking `Arrow` is considered the most complete FP library for kotlin.

Reply:
I get your point, however I do not think attendees (beginner to intermediate) will have time to dig in arrow implementation details, to find appropriate methods (like I unsuccessfully did).
We could, as an alternative, implement a simple validation mechanism gathering all field errors.

Reply:
I agree, thats why I added a comment containing useful functions. That might not be enough for beginners though (:sob:). I propose the following alternatives:

* Create another markdown (or even in the readme) giving more details on the kind of problem vs function to use. Some sections could include (applicatives, monads, disjunctions, etc)
* Improve the comment section in `UserValidation` and maybe a proper `Kdoc` showing how to use the functions (map2, tupled, left, right, etc)

Reply:
Actually I just added to the `kdoc` of the file